### ISSUE TITLE
[Feature] More configuration options

### ISSUE THREAD

**Issue by paz** [AnvilPowered/Catalyst#22]
I commented on this on the velocity forums, but the configuration leaves much to be desired.
eg: disabling certain features, such as chat. or functions of chat, such as nicknames, etc

An option I would really like is allowing to only use global chat with a prefix, like "!message here" would act as a shout.
A bit offtopic, but is there some way to share the nicknames with the other servers? eg with essentials

As much configuration as possible, in simpler terms. At the moment I only find use for the tablist, which i've been using a fork of your older plugin GlobalTab to accomplish.

**Comment by STG-Allen** [comment]
As for config options, I have plans on adding more once I finish finals. 

For the chat channels, I plan on re-working the current structure to allow for channels. Something to keep your eye out for. Also, what are you meaning with "Share nicknames". If you want to run catalyst on two seperate proxies, you can just connect Catalyst to the same Mongo database, using xodus won't work for multiple connections at once.

**Comment by paz** [comment]
I mean with spigot servers, because plugins use nickname on there. Is there any way to add a hook to replace essentials nicknames?

**Comment by STG-Allen** [comment]
I can look into this in the future, but the main function of the nicknames is to be used in the proxy chat itself. Adding this will require me to have a spigot side, which i am not necessarily opposed to, but it'll be lower on my list of features.

**Comment by STG-Allen** [comment]
This is what I have come up with thus far, let me know if there are more options you would like to see added. https://hastebin.com/dixebibaku.php

**Comment by paz** [comment]
option to disable discord, chat, motd/ping hover message etc, tab list, forced host MOTDs
just so you only have to use the modules you want to use

**Comment by STG-Allen** [comment]
![image](https://user-images.githubusercontent.com/26695048/81479320-74719200-91f0-11ea-9357-949597cca1e5.png)
As shown in the image above, you can disable the forced host motd's, discord and proxy-wide chat. I can add the option to disable the base motd+hover message/playername as well.

**Comment by paz** [comment]
oh my bad i didn't see that section somehow, yeah sounds good

**Comment by STG-Allen** [comment]
These additions have been made public in release 0.2.0
